docs/dev/build.md

```diff
@@ -102,82 +166,85 @@ chmod +x scripts/submodule_update_with_gitee.sh
 
-4. Before running the samples, add paths to the Threading Building Blocks (TBB) and OpenCV binaries used for
-   the build to the `%PATH%` environment variable. By default, TBB binaries are
-   downloaded by the CMake-based script to the `<openvino_repo>/temp/tbb/bin`
-   folder, OpenCV binaries to the `<openvino_repo>/temp/opencv_4.5.0/opencv/bin`
-   folder.
+4. Before running the samples, add paths to the Threading Building Blocks (TBB) binaries used for the build to the `%PATH%` environment variable. By default, TBB binaries are downloaded by the CMake-based script to the `<openvino>/temp/tbb/bin` folder.
 
 ### Additional Build Options
 
 - Internal JIT GEMM implementation is used by default.
 
-- Threading Building Blocks (TBB) is used by default. To build Inference
-  Engine with OpenMP threading, set the `-DTHREADING=OMP` option.
+- Threading Building Blocks (TBB) is used by default. To build Inference Engine with OpenMP threading, set the `-DTHREADING=OMP` option.
 
-- Required versions of TBB and OpenCV packages are downloaded automatically by
-  the CMake-based script. If you want to use the automatically-downloaded
-  packages but you have already installed TBB or OpenCV packages configured in
-  your environment, you may need to clean the `TBBROOT` and `OpenCV_DIR`
-  environment variables before running the `cmake` command; otherwise they won't
-  be downloaded and the build may fail if incompatible versions were installed.
+- Required versions of TBB and OpenCV packages are downloaded automatically by the CMake-based script. If you want to use the automatically-downloaded packages but you have already installed TBB or OpenCV packages configured in your environment, you may need to clean the `TBBROOT` and `OpenCV_DIR` environment variables before running the `cmake` command; otherwise they won'tnbe downloaded and the build may fail if incompatible versions were installed.
 
-- If the CMake-based build script can not find and download the OpenCV package
-  that is supported on your platform, or if you want to use a custom build of
-  the OpenCV library, refer to the 
-  [Use Custom OpenCV Builds](https://github.com/openvinotoolkit/openvino/wiki/CMakeOptionsForCustomCompilation#Building-with-custom-OpenCV)
-  section for details.
+- If the CMake-based build script can not find and download the OpenCV package that is supported on your platform, or if you want to use a custom build of the OpenCV library, refer to the [Use Custom OpenCV Builds](./cmake_options_for_custom_comiplation.md#Building-with-custom-OpenCV) section for details.
 
-- To switch off/on the CPU and GPU plugins, use the `cmake` options
-  `-DENABLE_INTEL_CPU=ON/OFF` and `-DENABLE_INTEL_GPU=ON/OFF` respectively.
+- To switch off/on the CPU and GPU plugins, use the `cmake` options `-DENABLE_INTEL_CPU=ON/OFF` and `-DENABLE_INTEL_GPU=ON/OFF` respectively.
```

Comment:
I think we don't need to have flags here... All these flags can be described in separate section

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2